homeassistant/components/lyric/manifest.json

```diff
@@ -22,5 +22,5 @@
   "iot_class": "cloud_polling",
   "loggers": ["aiolyric"],
   "quality_scale": "silver",
-  "requirements": ["aiolyric==1.0.9"]
+  "requirements": ["aiolyric==1.1.0"]
```

Comment:
This PR is also a dependency bump. Can you please create a separate PR for the dependency dump, where you please also include a diff of the library?

Please leave this PR in draft until the newly created dependency dump PR is merged.

Reply:
Sorry, first time contributing.  I have opened a separate pull request here.  https://github.com/home-assistant/core/pull/98113

Reply:
Switching back to draft due to comment on https://github.com/home-assistant/core/pull/98113.  It doesn't appear to be code I touched during my update, but I'll investigate and bump when fixed.